### PR TITLE
@sweir27 => [XAPP] Change default setting of xapp token constant to better support reloading

### PR DESCRIPTION
**Pull request by mzikherman**
This is a weird one! Basically, when you triggered a hot-reload of schema files, eventually the `config` file would get reloaded (I think). And there...the `GRAVITY_XAPP_TOKEN` is set to `null`. All subsequent unauthenticated requests would then fail (after any 60 second long cached data expired). Making authenticated requests wouldn't result in an error as sending an access token is sufficient.

That's pretty annoying! (and sort of breaks the benefit of hot reloading completely).

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -175,8 +176,9 @@ export default {
   GRAVITY_API_URL,
   GRAVITY_ID,
   GRAVITY_SECRET,
-  // Gets set from src/index.js
-  GRAVITY_XAPP_TOKEN: null as string | null,
+  // Gets set (+ refreshed) from src/index.js.
+  // Initially set from library for hot-reloading.
+  GRAVITY_XAPP_TOKEN: xapp.token,
```
Comment (mzikherman) [inline src/config.ts]: So, this is 'unsafe', since `xapp.token` isn't at all guaranteed to be present. However, once the app is booted, it will be present (and the boot process and refreshes overwrite it anyway).

Therefore, I think it's an improvement over the existing initialization, to set it from the library's value. It might still be undefined at load time, but it'll always have a value when it's actually used to satisfy a request. And by being set from the library, even hot reloads would maintain the value.

Basically, I think `xapp.token` is a saner default than an explicit `null` or `undefined`.
